### DISCUSSION TITLE
关于组件单元测试的方式

### DISCUSSION THREAD

**Discussion by zdxxxxxx** [youzan/vant#13050] [Q&A]
在根目录运行`pnpm  test` ，可以完成全部组件的单元测试。但怎么跑单个组件的单元测试呢？
方法1：
在/packages/vant/目录下，运行 `pnpm test ./src/button`

方法2：
在根目录运行`pnpm test ./src/button`

通常开发组件的时候，不会cd到/packages/vant/目录下，而是在根目录执行命令的时候比较多。
但使用方法2，后面的路径参数`./src/button`，是反直觉的，他并不是一个合法路径。

这里有什么更优雅的方式吗？仅讨论

**Comment by inottn** [comment]
pnpm test /button/
